packages/ListBox/src/components/Filter/Filter.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useListBox from "../../useListBox";
+import * as effects from "./effects";
+import { filter, applyFilter } from "./helpers";
+import { FilterContainerStyled, FilterInputStyled, FilterSearchIconStyled } from "./Filter.styles";
+
+const propTypes = {
+  filter: PropTypes.func,
+  filterExcludeSelectedOptions: PropTypes.bool,
+  hasSearchIcon: PropTypes.bool,
+  noResultsMessage: PropTypes.string,
+  onChangeFilter: PropTypes.func,
+  onKeyDown: PropTypes.func,
+  placeholder: PropTypes.string,
+  renderFilter: PropTypes.func,
+  value: PropTypes.string,
+};
+
+const defaultProps = {
+  filter: null,
+  filterExcludeSelectedOptions: false,
+  hasSearchIcon: true,
+  onChangeFilter: null,
+  onKeyDown: null,
+  placeholder: "Filter...",
+  noResultsMessage: "Your search did not match any options.",
```

Comment:
// TODO: use L10n

Reply:
^ what do you mean? 🤔

Reply:
Yeah, you probably have to set the default to `null`, then use the L10nized string when rendering, if it's still `null`.